CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.10.2] - 2024-09-03
+
+### Added
+
+- `http_challenge` NodeAttestor plugin (#4909)
+- Experimental support for validating container image signatures through Sigstore selectors in the docker Workload Attestor (#5272)
+- Metrics to track event-based cache (#5411)
```

Comment:
```suggestion
- Metrics that help monitoring the event-based cache (#5411)
```

Reply:
Maybe "Metrics that help monitor the" or "Metrics for monitoring the"?